web/app/css/tap.css

```diff
@@ -18,3 +18,20 @@
     color: rgba(52, 137, 206, 0.906);
   }
 }
+
+.tap-more-info {
+  padding-right: 50px;
+
+  & .tap-info-section {
+    padding: 24px 0;
+
+    & hr {
+      color: red;
+
```

Comment:
the `hr` elements don't quite extend to the width of the text. what do you think about something like:
`padding: 0;`

![screen shot 2018-09-13 at 4 44 18 pm](https://user-images.githubusercontent.com/236915/45521775-60079880-b774-11e8-8945-299e8ef8fefe.png)

i noticed the mocks have the `hr`'s going all the way across, that looks nice too.

Reply:
yeahhh the reason I have that padding there is because of the + / - that we're using from the ant expandable table row, which causes this expanded section to be 50px from the left. 
We could accordion all these rows, but for now I've just made the padding symmetric. We're gonna move off the ant tables soon so it won't matter than either!